Analysis/bin/ShuffleMergeSpectral.cxx

```diff
@@ -398,6 +404,7 @@ class SpectrumHists {
           double dis_scale = std::min(1.0, exp_disbalance*
                              ttype_entries.at(type)->Integral(0,binNx,binNy-1,binNy)*
                              ttype_prob.at(type)->GetBinContent(x,y));
+          if(dis_scale<=0.0) dis_scale = 1.0;
```

Comment:
I think this remark indeed makes sense, I agree, however I am looking at it from the following point of view: it can occur only in case when ttype_entries.at(type)->Integral(0,binNx,binNy-1,binNy) == 0, in this case our required disbalance is simply unreachable, so maybe we should raise an exception? In the opposite case, user will have what he did not expected from his input arguments.

Reply:
I agree, better to throw an exception, because integral=0 is an unexpected input.

Reply:
Thanks for your comments! I made the changes you asked for.